clover/__init__.py

```diff
@@ -67,6 +67,8 @@ def count_loc(self, sources):
             # Old version of coverage
             sources = [""]
         for source in sources:
+            if not source:
+                continue
```

Comment:
`for source in filter(None, sources)`? This won't change lines count and break tests

Reply:
right, a user of you tool, that fix the error above and so I've shared it with you hoping that helps.
Here what I have:

docker "python:3.8-slim"

pytest==6.2.2
pytest-cov==2.11.1
coverage2clover==3.2.0
coverage==5.5
...

when I'll find more time I'll try to better understand what your code is doing :)

Reply:
note: I can see many FAILED building this simple Dockerfile:

```
FROM python:3.8-slim

RUN apt-get update && \
    apt-get -y install git python3-pip

RUN pip install --upgrade pip

RUN pip install tox && \
    git clone https://github.com/tumb1er/coverage2clover /coverage2clover

ENV PYTHONPATH /coverage2clover

WORKDIR /coverage2clover

RUN pip install -r requirements.txt
RUN tox -e py38
```

do you know why?

Reply:
Yes, I know. This is because coverage file is generated from coverage2clover tests dynamically, so when you change code, coverage file is changing. It's a mistake of youth, but there is no plans to change it in near future.

Reply:
well, sorry for the delay I'm very busy. Here is how to replicate the error:

myapp.py
```

class MyApp:

    def myfunction(self, param):
        if param == "one":
            return 1
        elif param == "two":
            return 2
        else:
            return -1

```

test_myapp.py
```
import unittest
from app.myapp import MyApp


class TestMyApp(unittest.TestCase):

    def test_myfunction(self):
        a = MyApp()

        self.assertEqual(a.myfunction("one"), 1)
        self.assertEqual(a.myfunction("ggg9-gb"), -1)

```

Dockerfile
```
FROM python:3.8-slim

RUN apt-get update && \
    apt-get -y install git python3-pip

RUN pip install --upgrade pip
RUN pip install coverage2clover pytest pytest-cov

ENV PYTHONPATH /src

WORKDIR /src

RUN mkdir -p /src/app && mkdir -p /src/tests && mkdir /my-reports-dir && touch /src/app/__init__.py

COPY myapp.py /src/app/
COPY test_myapp.py /src/tests

RUN cd /my-reports-dir \
    && pytest --cov=app --cov-report=xml --junitxml=results.xml /src/tests \
    && coverage2clover -i coverage.xml -o clover.xml
```

create those 3 files in the same directory, then just build it:
```
docker build -t "test_coverage2clover:latest" .
```

Reply:
note that I'm executing the commands from **/my-reports-dir**  not from **/src** 

my guess: **pytest** understand **PYTHONPATH** but your script doesn't

Reply:
Can you provide a `coverage.xml` file?

Reply:
I might have been unclear, the docker build of my previous comment provides everything you need and I guess you know how use docker. 
Do you want me to extract that coverage.xml for you or are you asking for the one produced by my project?